### ISSUE TITLE
Some of mochaOpts don't seem to work in wdio.conf.js

### ISSUE THREAD

**Issue by silvenon** [webdriverio/webdriverio#660]
I tried passing these:

```js
   mochaOpts: {
        ui: 'bdd',
        compilers: ['js:babel-core/register'],
        require: ['./test/helpers/common.js']
    },
```

and running `wdio wdio.conf.js`, but it didn't understand the `import` command. When I replace `import`s with `require`s, it didn't require `common.js` so tests failed again, because they relied on it. I'm not sure which Mocha options are supported.

**Comment by christian-bromann** [comment]
According to [the docs](https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options) this was missing when using Mocha programmatically. You could basically just import them in the before hook but I've added that to the internal Mocha framework to prevent confusion.

**Comment by christian-bromann** [comment]
Fixed and published with `v3.2.0`

**Comment by silvenon** [comment]
Awesome! Thanks :+1:
